prompting/base/validator.py

```diff
@@ -304,7 +304,7 @@ def update_scores(self, rewards: np.ndarray, uids: list[int]):
         # Compute forward pass rewards, assumes uids are mutually exclusive.
         # shape: [ metagraph.n ]
         step_rewards = np.copy(self.scores)
-        step_rewards[uids] = rewards
+        step_rewards[np.array(uids).astype(int)] = rewards
```

Comment:
Why do we need explicit casting?

Reply:
there was some place where it had the uids as floats... obviously we should fix it in the place where it happens, but with us wanting to release today I didn't have the time to figure out where it is